src/ORM/Filters/ExactMatchFilter.php

```diff
@@ -229,12 +227,12 @@ public function isEmpty()
      * - The values being filtered are all either integers or valid integer strings
      * - Using placeholders for integer ids has been configured off
      *
-     * Putting IDs directly into a where clause instead of using placehodlers was measured to be significantly
+     * Putting IDs directly into a where clause instead of using placeholders was measured to be significantly
      * faster when querying a large number of IDs e.g. over 1000
      */
     private function usePlaceholders(string $column, array $values): bool
     {
-        if ($this->config()->get('use_placeholders_for_integer_ids')) {
+        if (Config::inst()->get(DataList::class, 'use_placeholders_for_integer_ids')) {
```

Comment:
```suggestion
        if (DataList::config()->get('use_placeholders_for_integer_ids')) {
```
DataList has the Configurable trait through inheritance, so we can use the shorter syntax here.

Reply:
Updated